src/views/enrollUser/EnrollUserForm.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/RegistrationFormFactory',
+  'models/ProfileSchema',
+], function (Okta, RegistrationFormFactory, ProfileSchema) {
+  return Okta.Form.extend({
+    layout: 'o-form-theme',
+    autoSave: true,
+    noCancelButton: true,
+    title: Okta.loc('registration.form.title', 'login'),
+    save: Okta.loc('registration.form.submit', 'login'),
+    initialize: function (options) {
+      this.options = options || {};
+      this.schema =
+      new ProfileSchema({ profileSchemaAttributes: this.options.appState.get('policy').registration.profile });
```

Comment:
the indent looks incorrect. no linting error?

Reply:
hmmm, I din't get any lint errors

Reply:
@haishengwu-okta I don't have a strong opinion. I could add a es lint rule to allow it on the same line or use a variable `profile` to hold `this.options.appState.get('policy').registration.profile`.

Reply:
it's fine to keep it for now. might be lint rule config inappropriately.